### PR TITLE
Rename CropManager => Cropper

### PR DESCRIPTION
**Pull request by talbaneth**

### CODE REVIEW

**Review by hexonaut — APPROVED** [review]
LGTM

File: src/test/Cropper-unit.t.sol [hunk 1]
```diff
@@ -18,39 +18,39 @@ pragma solidity 0.6.12;
...
 import {CropJoin, CropJoinImp} from "../CropJoin.sol";
-import "../CropManager.sol";
+import "../Cropper.sol";
 import {MockVat} from "./CropJoin-unit.t.sol";
 
 contract Usr {
 
     CropJoinImp    adapter;
-    CropManagerImp manager;
+    CropperImp cropper;
```
Comment (gbalabasquer) [inline src/test/Cropper-unit.t.sol:27]: we might want to align it with the value above as it was before.

File: src/test/LidoJoin-integration.t.sol [hunk 2]
```diff
@@ -182,16 +182,16 @@ contract LidoIntegrationTest is TestBase {
...
         join = SynthetixJoinImp(address(baseJoin));
         join.init();
-        CropManager baseManager = new CropManager();
-        baseManager.setImplementation(address(new CropManagerImp(address(vat))));
-        manager = CropManagerImp(address(baseManager));
-        baseJoin.rely(address(manager));
-        baseJoin.deny(address(this));    // Only access should be through manager
+        Cropper baseManager = new Cropper();
+        baseManager.setImplementation(address(new CropperImp(address(vat))));
+        cropper = CropperImp(address(baseManager));
```
Comment (gbalabasquer) [inline src/test/LidoJoin-integration.t.sol:187]:
```suggestion
        Cropper baseCropper = new Cropper();
        baseCropper.setImplementation(address(new CropperImp(address(vat))));
        cropper = CropperImp(address(baseCropper));
```

File: src/test/SushiJoinV1-integration.t.sol [hunk 3]
```diff
@@ -220,17 +220,17 @@ contract SushiV1IntegrationTest is TestBase {
...
         join = SushiJoinImp(address(baseJoin));
         join.initApproval();
-        CropManager baseManager = new CropManager();
-        baseManager.setImplementation(address(new CropManagerImp(address(vat))));
-        manager = CropManagerImp(address(baseManager));
-        baseJoin.rely(address(manager));
-        baseJoin.deny(address(this));    // Only access should be through manager
+        Cropper baseManager = new Cropper();
+        baseManager.setImplementation(address(new CropperImp(address(vat))));
+        cropper = CropperImp(address(baseManager));
```
Comment (gbalabasquer) [inline src/test/SushiJoinV1-integration.t.sol:225]:
```suggestion
        Cropper baseCropper = new Cropper();
        baseCropper.setImplementation(address(new CropperImp(address(vat))));
        cropper = CropperImp(address(baseCropper));
```

File: src/test/SushiJoinV2-integration.t.sol [hunk 4]
```diff
@@ -239,18 +239,18 @@ contract SushiIntegrationTest is TestBase {
...
         join = SushiJoinImp(address(baseJoin));
         join.initApproval();
-        CropManager baseManager = new CropManager();
-        baseManager.setImplementation(address(new CropManagerImp(address(vat))));
-        manager = CropManagerImp(address(baseManager));
-        baseJoin.rely(address(manager));
-        baseJoin.deny(address(this));    // Only access should be through manager
+        Cropper baseManager = new Cropper();
+        baseManager.setImplementation(address(new CropperImp(address(vat))));
+        cropper = CropperImp(address(baseManager));
```
Comment (gbalabasquer) [inline src/test/SushiJoinV2-integration.t.sol:244]:
```suggestion
        Cropper baseCropper = new Cropper();
        baseCropper.setImplementation(address(new CropperImp(address(vat))));
        cropper = CropperImp(address(baseCropper));
```

File: src/test/Cropper-unit.t.sol (hunk 1 above)
Comment (gbalabasquer) [inline src/test/Cropper-unit.t.sol:27]: Minor thing: tabulation between values.

Comment (talbaneth) [reply src/test/Cropper-unit.t.sol]: Done.

**Review by gbalabasquer — APPROVED** [review]
LGTM

**Pull request merged**